packages/frontend/src/pages/RestoreAccount/ui/CheckIcon.tsx

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+
+const CheckIcon = () => (
```

Comment:
This icon seems pretty common, we can move it to `components/svg/`

Reply:
fixed